README.md

```diff
@@ -32,47 +32,138 @@ Also the web service itself is open source.
 
 The frontend is implemented using React and the backend is built on top of FastAPI and Pydantic.
 
+## Quick Start for Developers
+
```

Comment:
IMO this should just be a link to the separate developer documentation. The following lines are redundant with what I already reviewed.